### PR TITLE
Initial Guide XAMPP (DE)

### PR DESCRIPTION
**Pull request by fkaminski**
See Issue #685
First German Commit. Translation (EN) after review(s).
Thanks

### CODE REVIEW

**Comment by fkaminski** [comment]
Add commit with EN translation (via deepl).

File: docs/manual/guides/local-installation/xampp.de.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+»xampp_shell.bat« erstellt und eine entsprechende »XAMPP Konsole« geöffnet. Zwecks Überprüfung gibts du in dieser 
+Konsole den Befehl `php -v` ein. Als Ausgabe solltest du die entsprechende PHP (CLI) Version erhalten und kannst
+die Konsole dann schliessen. 
+
+In deinem Browser erreichst du über die Eingabe von `http://localhost` das 
+»XAMPP Dashboard« mit generellen Informationen. Hier findest du im oberen Menü u. a. einen Link `PHPInfo` mit 
+Informationen zur aktuellen PHP-Konfiguration.
+
+{{% notice info %}}
+Du solltest das "XAMPP Control-Panel" (xampp-control.exe) und die »XAMPP Konsole« (xampp_shell.bat) immer mit
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Du solltest das »XAMPP Control-Panel« (xampp-control.exe) und die »XAMPP Konsole« (xampp_shell.bat) immer mit 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+»XAMPP Dashboard« mit generellen Informationen. Hier findest du im oberen Menü u. a. einen Link `PHPInfo` mit 
+Informationen zur aktuellen PHP-Konfiguration.
+
+{{% notice info %}}
+Du solltest das "XAMPP Control-Panel" (xampp-control.exe) und die »XAMPP Konsole« (xampp_shell.bat) immer mit 
+Administrator Rechten starten.
+{{% /notice %}}
+
+
+## Die PHP Konfiguration
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
## Die PHP-Konfiguration
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+| Eintrag                 | Änderung                   |
+|:------------------------|:---------------------------|
+| **memory_limit = 512M** | **memory_limit = -1**      |
+| **;extension=intl**     | **extension=intl**         |
+
+{{% notice tip %}}
+Die genannten Änderungen sind für die Installation über den [PHP-Composer](https://getcomposer.org/) oder über den 
+[Contao-Manager](/de/installation/contao-manager/) zwingend notwendig. 
+Darüber hinaus könnstest du optional auch die Einträge »allow_url_fopen«, »max_execution_time«
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Darüber hinaus könntest du optional auch die Einträge »allow_url_fopen«, »max_execution_time« 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
+{{% notice tip %}}
+Die genannten Änderungen sind für die Installation über den [PHP-Composer](https://getcomposer.org/) oder über den 
+[Contao-Manager](/de/installation/contao-manager/) zwingend notwendig. 
+Darüber hinaus könnstest du optional auch die Einträge »allow_url_fopen«, »max_execution_time« 
+oder »file_uploads« überprüfen und anpassen.
+{{% /notice %}}
+
+**Herzlichen Glückwunsch!** 
+Du hast alle Vorbereitungen für eine lokale Contao Installation abgeschlossen.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Du hast alle Vorbereitungen für eine lokale Installation von Contao abgeschlossen.
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+
+## Lokale PHP-Composer Installation
+
+Wir haben keine »XAMPP» Installation unter Windows im eigentlichen Sinne durchgeführt. Daher werden wir die benötigte 
+PHP-Composer Datei ebenfalls lediglich kopieren.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
PHP-Composer-Datei ebenfalls lediglich kopieren. 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+```
+
+In deinem XAMPP-Verzeichnis befindet sich nun die Datei `composer.phar`. Mit Eingabe von `php composer.phar -V` solltest 
+du eine entsprechende Rückmeldung erhalten.
+
+
+### Contao mit PHP-Composer installieren
+
+Ohne weitere vHost-Konfiguration (s. u.) erwartet XAMPP die Dateien deiner Webseite im XAMPP-Verzeichnis `htdocs`. 
+In der »XAMPP-Konsole« wechselst du daher in dieses Unterverzeichnis und installierst eine Contao Version
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
In der »XAMPP-Konsole« wechselst du daher in dieses Unterverzeichnis und installierst eine Contao-Version 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+
+Ohne weitere vHost-Konfiguration (s. u.) erwartet XAMPP die Dateien deiner Webseite im XAMPP-Verzeichnis `htdocs`. 
+In der »XAMPP-Konsole« wechselst du daher in dieses Unterverzeichnis und installierst eine Contao Version 
+mit Eingabe von z. B.:
+
+```php
+php ../composer.phar create-project contao/managed-edition demo 4.9
+```
+
+Deine Contao Installation befindet sich im Anschluss im XAMPP-Verzeichnis `htdocs\demo`.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Deine Contao-Installation befindet sich im Anschluss im XAMPP-Verzeichnis `htdocs\demo`.
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+
+{{% notice note %}}
+Diese Vorgehensweise ist optional. Du könntest die Contao Installation auch direkt über den Contao-Manager ausführen.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Optional könntest du die Installation von Contao auch direkt über den Contao-Manager ausführen.
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+
+
+## Contao Aufrufe (ohne eigenen vHost)
+
+Im »XAMPP Control-Panel« kannst du im Bereich »MySql« über die Schaltfläche »Admin« `phpMyAdmin` ausführen und wie 
+gewohnt eine Datenbank anlegen. Sofern noch nicht geschehen kopierst du den 
+[Contao-Manager](https://contao.org/de/download.html) z. B. in das Verzeichnis `demo\web` und benennst die Datei danach 
+in »contao-manager.phar.php« um. 
+
+Deine Contao Installation kann jetzt abgeschlossen werden. Allerdings unterscheiden sich die Contao Aufrufe, ohne
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Deine Installation kann jetzt abgeschlossen werden. Allerdings unterscheiden sich die Contao-Aufrufe, ohne
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 10]
```diff
@@ -0,0 +1,230 @@
...
+
+{{% notice note %}}
+Diese Vorgehensweise ist optional. Du könntest die Contao Installation auch direkt über den Contao-Manager ausführen.
+Erstelle dir hierzu besipielsweise im XAMPP-Verzeichnis `htdocs` die Verzeichnisse `demo\web`. Kopiere den 
+[Contao-Manager](https://contao.org/de/download.html) in das Verzeichnis `demo\web` und benenne die Datei danach 
+in »contao-manager.phar.php« um.
+{{% /notice %}}
+
+
+## Contao Aufrufe (ohne eigenen vHost)
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
## Contao-Aufrufe (ohne eigenen vHost)
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 11]
```diff
@@ -0,0 +1,230 @@
...
+In der Datei `D:\xampp\apache\php.ini` suchst du die Zeilen »memory_limit« und »extension=intl« und änderst diese wie folgt:
+
+| Eintrag                 | Änderung                   |
+|:------------------------|:---------------------------|
+| **memory_limit = 512M** | **memory_limit = -1**      |
+| **;extension=intl**     | **extension=intl**         |
+
+{{% notice tip %}}
+Die genannten Änderungen sind für die Installation über den [PHP-Composer](https://getcomposer.org/) oder über den 
+[Contao-Manager](/de/installation/contao-manager/) zwingend notwendig.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
[Contao Manager](/de/installation/contao-manager/) zwingend notwendig. 
```

File: docs/manual/guides/local-installation/xampp.de.md (hunk 8 above)
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Diese Vorgehensweise ist optional. Du könntest die Installation auch direkt über den Contao Manager ausführen.
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 12]
```diff
@@ -0,0 +1,230 @@
...
+[Contao-Manager](https://contao.org/de/download.html) in das Verzeichnis `demo\web` und benenne die Datei danach 
+in »contao-manager.phar.php« um.
+{{% /notice %}}
+
+
+## Contao Aufrufe (ohne eigenen vHost)
+
+Im »XAMPP Control-Panel« kannst du im Bereich »MySql« über die Schaltfläche »Admin« `phpMyAdmin` ausführen und wie 
+gewohnt eine Datenbank anlegen. Sofern noch nicht geschehen kopierst du den 
+[Contao-Manager](https://contao.org/de/download.html) z. B. in das Verzeichnis `demo\web` und benennst die Datei danach
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
[Contao Manager](https://contao.org/de/download.html) z. B. in das Verzeichnis `demo\web` und benennst die Datei danach 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 13]
```diff
@@ -0,0 +1,230 @@
...
+| Ziel            | Aufruf                                                                                                       |
+|:----------------|:-------------------------------------------------------------------------------------------------------------|
+| Install-Tool    | http://localhost/demo/web/contao/install                                                                     |
+| Contao-Backend  | http://localhost/demo/web/contao                                                                             |
+| Contao-Frontend | http://localhost/demo/web                                                                                    |
+| Contao-Manager  | http://localhost/demo/web/contao-manager.phar.php/ <br/>**Der Schrägstrich am Ende ist zwingend notwendig!** |
+
+{{% notice warning %}}
+Du kannst demnach ohne eigene »XAMPP vHost-Konfiguration« (s. u.) mit Contao arbeiten, solange du die obigen Aufrufe
+berücksichtigst. Innerhalb der Contao eigenen Verlinkung (z. B.: Den Contao-Manager über das Backend starten usw.)
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
berücksichtigst. Innerhalb der Contao eigenen Verlinkung (z. B.: Den Contao Manager über das Backend starten usw.) 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 14]
```diff
@@ -0,0 +1,230 @@
...
+|:----------------|:-------------------------------------------------------------------------------------------------------------|
+| Install-Tool    | http://localhost/demo/web/contao/install                                                                     |
+| Contao-Backend  | http://localhost/demo/web/contao                                                                             |
+| Contao-Frontend | http://localhost/demo/web                                                                                    |
+| Contao-Manager  | http://localhost/demo/web/contao-manager.phar.php/ <br/>**Der Schrägstrich am Ende ist zwingend notwendig!** |
+
+{{% notice warning %}}
+Du kannst demnach ohne eigene »XAMPP vHost-Konfiguration« (s. u.) mit Contao arbeiten, solange du die obigen Aufrufe
+berücksichtigst. Innerhalb der Contao eigenen Verlinkung (z. B.: Den Contao-Manager über das Backend starten usw.) 
+erhälst du weiterhin Fehlermeldungen, da diese von einer »korrekten« Umgebung ausgehen. Empfehlenswert wäre daher die
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
erhältst du weiterhin Fehlermeldungen, da diese von einer »korrekten« Umgebung ausgehen. Empfehlenswert wäre daher die
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 15]
```diff
@@ -0,0 +1,230 @@
...
+weitere, einmalige XAMPP-Konfiguration über einen eigenen vHost.
+{{% /notice %}}
+
+
+## Eigene vHost-Konfiguration
+
+Eine einmalige, eigene XAMPP vHost-Konfiguration bietet zahlreiche Vorteile. Einerseits kannst du hierüber deine 
+Contao Installation(en) außerhalb des XAMPP-Verzeichnisses pflegen und andererseits diese mit eigenen, lokalen
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Contao-Installation(en) außerhalb des XAMPP-Verzeichnisses pflegen und andererseits diese mit eigenen, lokalen
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 16]
```diff
@@ -0,0 +1,230 @@
...
+Domain Namen verknüpfen.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Domain-Namen verknüpfen. 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 17]
```diff
@@ -0,0 +1,230 @@
...
+{{% /notice %}}
+
+
+## Eigene vHost-Konfiguration
+
+Eine einmalige, eigene XAMPP vHost-Konfiguration bietet zahlreiche Vorteile. Einerseits kannst du hierüber deine 
+Contao Installation(en) außerhalb des XAMPP-Verzeichnisses pflegen und andererseits diese mit eigenen, lokalen
+Domain Namen verknüpfen. 
+
+Erstelle dir ein neues Verzeichnis z. B. `D:\vhost\demo\web`. Kopiere den [Contao-Manager](https://contao.org/de/download.html)
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Erstelle dir ein neues Verzeichnis z. B. `D:\vhost\demo\web`. Kopiere den [Contao Manager](https://contao.org/de/download.html) 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 18]
```diff
@@ -0,0 +1,230 @@
...
+## Eigene vHost-Konfiguration
+
+Eine einmalige, eigene XAMPP vHost-Konfiguration bietet zahlreiche Vorteile. Einerseits kannst du hierüber deine 
+Contao Installation(en) außerhalb des XAMPP-Verzeichnisses pflegen und andererseits diese mit eigenen, lokalen
+Domain Namen verknüpfen. 
+
+Erstelle dir ein neues Verzeichnis z. B. `D:\vhost\demo\web`. Kopiere den [Contao-Manager](https://contao.org/de/download.html) 
+in dieses Verzeichnis und benenne die Datei danach in »contao-manager.phar.php« um. 
+
+### Domain Namen erstellen
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
### Domain-Namen erstellen
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 19]
```diff
@@ -0,0 +1,230 @@
...
+
+Wir möchten unsere Contao Installation über die Domain `demo.local` erreichen. Im Windows Verzeichnis `System32/drivers/etc`
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Wir möchten unsere Contao-Installation über die Domain `demo.local` erreichen. Im Windows Verzeichnis `System32/drivers/etc`
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 20]
```diff
@@ -0,0 +1,230 @@
...
+    AllowOverride All
+    Require all granted
+  </Directory>
+
+  ErrorLog "D:\xampp\apache\logs\demo.local_error.log"
+  CustomLog "D:\xampp\apache\logs\demo.local_access.log" combined
+</VirtualHost>
+```
+
+Passe hier nötigenfalls die Pfadangaben deinen aktuellen Verzeichnisse entsprechend an. Im »XAMPP Control-Panel«
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Passe hier nötigenfalls die Pfadangaben deiner aktuellen Verzeichnisse entsprechend an. Im »XAMPP Control-Panel« 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 21]
```diff
@@ -0,0 +1,230 @@
...
+    Require all granted
+  </Directory>
+
+  ErrorLog "D:\xampp\apache\logs\demo.local_error.log"
+  CustomLog "D:\xampp\apache\logs\demo.local_access.log" combined
+</VirtualHost>
+```
+
+Passe hier nötigenfalls die Pfadangaben deinen aktuellen Verzeichnisse entsprechend an. Im »XAMPP Control-Panel« 
+musst du die Module »Apache« und »MySQL« neu starten. Anschließend kannst du den Contao-Manager über den Aufruf
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
musst du die Module »Apache« und »MySQL« neu starten. Anschließend kannst du den Contao Manager über den Aufruf
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 22]
```diff
@@ -0,0 +1,230 @@
...
+`http://demo.local/contao-manager.phar.php` erreichen.
+
+
+{{% notice tip %}}
+Mit dieser Vorgehensweise könntest du auch mehrere Contao Installationen über verschiedene, lokale Domain Namen mit
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Mit dieser Vorgehensweise könntest du auch mehrere Contao-Installationen über verschiedene, lokale Domain-Namen mit 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 23]
```diff
@@ -0,0 +1,230 @@
...
+
+## SSL-Zertifikat erstellen
+
+Über den Aufruf von z. B.: `http://demo.local/contao-manager.phar.php` erhälst du im Contao-Manager einen Hinweis 
+das es sich um eine »Unsichere Verbindung« handelt. Dies ist korrekt, da XAMPP kein SSL-Zertifikat für unsere Domain
+`demo.local` ausliefert bzw. dieses nicht existiert.
+
+Im XAMPP-Verzeichnis `\apache` findest du die Datei »makecert.bat«, über die du dir ein neues, lokales V1-Zertifikat 
+erstellen kannst. Starte in der »XAMPP-Konsole» diese Datei und folge den Anweisungen. Deine Angaben sind hier 
+grundsätzlich beliebig. Lediglich die Angabe bei `Common Name` muss dem aktuellen, lokalen Domain Namen entsprechen.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
grundsätzlich beliebig. Lediglich die Angabe bei `Common Name` muss dem aktuellen, lokalen Domain-Namen entsprechen. 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 24]
```diff
@@ -0,0 +1,230 @@
...
+Über den Aufruf von z. B.: `http://demo.local/contao-manager.phar.php` erhälst du im Contao-Manager einen Hinweis 
+das es sich um eine »Unsichere Verbindung« handelt. Dies ist korrekt, da XAMPP kein SSL-Zertifikat für unsere Domain
+`demo.local` ausliefert bzw. dieses nicht existiert.
+
+Im XAMPP-Verzeichnis `\apache` findest du die Datei »makecert.bat«, über die du dir ein neues, lokales V1-Zertifikat 
+erstellen kannst. Starte in der »XAMPP-Konsole» diese Datei und folge den Anweisungen. Deine Angaben sind hier 
+grundsätzlich beliebig. Lediglich die Angabe bei `Common Name` muss dem aktuellen, lokalen Domain Namen entsprechen. 
+Für unser Beispiel also zwingend `demo.local`.
+
+In den Verzeichnissen `apache\conf\ssl.crt`, `apache\conf\ssl.csr` und `apache\conf\ssl.key` wurden den Angaben
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
In den Dateien `apache\conf\ssl.crt`, `apache\conf\ssl.csr` und `apache\conf\ssl.key` wurden den Angaben 
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 25]
```diff
@@ -0,0 +1,230 @@
...
+    Require all granted
+  </Directory>
+
+  ErrorLog "D:\xampp\apache\logs\demo.local_error.log"
+  CustomLog "D:\xampp\apache\logs\demo.local_access.log" combined
+</VirtualHost>
+```
+
+Ändere auch hier wieder die Pfade entsprechend deiner aktuellen Umsetzung und starte im »XAMPP Control-Panel« 
+die Module »Apache« und »MySQL« erneut. Anschließend kannst du den Contao-Manager über `https://demo.local/contao-manager.phar`
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
die Module »Apache« und »MySQL« erneut. Anschließend kannst du den Contao Manager über `https://demo.local/contao-manager.phar.php`
```

File: docs/manual/guides/local-installation/xampp.de.md [hunk 26]
```diff
@@ -0,0 +1,230 @@
...
+erreichen.
+
+Dein Browser wird dir einen Warnhinweis ausgeben, da dieser unser lokal erstelltes Zertifikat zwar erkennt aber 
+zunächst nicht vertraut. Dies kannst du aber ohne weiteres ignorieren und im Browser die Domain `demo.local` 
+als Ausnahme bestätigen.
+
+Im Contao-Manager selbst wird fortan der vorherige Hinweis auf eine »Unsichere Verbindung« nicht mehr erfolgen.
```
Comment (netzarbeiter) [inline docs/manual/guides/local-installation/xampp.de.md]:
```suggestion
Im Contao Manager selbst wird fortan der vorherige Hinweis auf eine »Unsichere Verbindung« nicht mehr angezeigt.
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Comment by fkaminski** [comment]
Thank you

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**